src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services13.php

```diff
@@ -30,6 +30,9 @@ public function __construct()
         $this->methodMap = array(
             'bar' => 'getBarService',
         );
+        $this->privates = array(
+            'service_container' => true,
```

Comment:
Could be left out indeed, but im not sure if we care :) as it requires adding code; which defeats this PR :)